docs/usage/self-hosted-experimental.md

```diff
@@ -71,14 +71,18 @@ You can set the config file Renovate should read with the `RENOVATE_CONFIG_FILE`
 
 The process that runs Renovate must have the correct permissions to delete the config file.
 
+## `RENOVATE_X_DOCKER_HUB_TAGS_DISABLE`
+
+If set to any value, Renovate will stop using the Docker Hub API (`https://hub.docker.com`) to fetch tags and instead use the normal Docker API for images pulled from `https://index.docker.io`.
+
 ## `RENOVATE_X_DOCKER_MAX_PAGES`
 
 If set to an integer, Renovate will use this as max page number for docker tags lookup on docker registries, instead of the default 20 pages.
 This is useful for registries which ignores the `n` parameter in the query string and only return 50 tags per page.
 
-## `RENOVATE_X_EAGER_GLOBAL_EXTENDS`
+## `RENOVATE_X_EXEC_GPID_HANDLE`
 
-Resolve and merge `globalExtends` presets before other global config, instead of after.
+If set, Renovate will terminate the whole process group of a terminated child process spawned by Renovate.
```

Comment:
How can you terminate something that's already terminated?

This needs a rewrite to explain more clearly what's going on.

I think you mean something like this:

> When a child process dies, Renovate will kill the parent process as well.

But in programmer words. :upside_down_face:

Reply:
I think this one's alright.